api/src/main/resources/org/openmrs/module/mirebalaisreports/sql/mexicoCesMeds.sql

```diff
@@ -0,0 +1,37 @@
+SELECT
+    MAX(d.name) "Medicamento",
+    SUM(disp_quant.value_numeric) "Cantidad"
+FROM obs disp_quant
+         LEFT JOIN obs disp_drug
```

Comment:
Not 100% sure about the left join here.  Is there a valid case where you'd want to return quantity without the drug?

Reply:
I think you're right; there isn't